app/models/trading_partner.rb

```diff
@@ -3,6 +3,7 @@ class TradingPartner
   include ActiveModel::Attributes
 
   attribute :country, :string
+  attribute :anchor, :string
```

Comment:
AFAIK, this attribute is never used?

Reply:
If I don't set this I get attribute errors